cms/envs/bok_choy.env.json

```diff
@@ -103,5 +104,12 @@
     "TECH_SUPPORT_EMAIL": "technical@example.com",
     "THEME_NAME": "",
     "TIME_ZONE": "America/New_York",
-    "WIKI_ENABLED": true
+    "WIKI_ENABLED": true,
+    "THIRD_PARTY_AUTH_BACKENDS": [
```

Comment:
Are any of the backends in this list actually used in bok_choy tests other than the dummy backend?